### DISCUSSION TITLE
Create Folders Within Projects

### DISCUSSION THREAD

**Discussion by Thoomas247** [premake/premake-core#2162] [Q&A]
Is there a way to create arbitrary folders using premake? For example, in every project I generate, I want to add an empty subfolder called src.

**Comment by nickclark2016** [comment]
This should be possible at the time which you run Premake.  You should be able to use `os.mkdir` to create an arbitrary folder, you would just need to point it where you want it.

https://premake.github.io/docs/os.mkdir/
